index.js

```diff
@@ -3,13 +3,7 @@ var spawn = require('child_process').spawn;
 var path = require('path');
 var format = require('util').format;
 var Configstore = require('configstore');
```

Comment:
I would also inline `configstore`. From earlier measurements I've found it to be slow to load too.